<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:0.1em solid rgb(0, 0, 0);
/*
border-radius:5px;
*/
background-color: rgba(200, 200, 200,0);
border-style: outset;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tea_class">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:0.1em solid rgb(0, 0, 0);
/*
border-radius:5px;
*/
background-color: rgba(200, 200, 200,0);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tea_password">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="stuRadio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;
color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);
</string>
   </property>
   <property name="text">
    <string>我是学生</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_class">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tea_name">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="teacherRadio">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;
color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);
</string>
   </property>
   <property name="text">
    <string>我是教师</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_id">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
border:2px solid rgb(200, 200, 200);
border-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
background-color: rgba(0%,0%,0%,30%);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
/*
border:0.1em solid rgb(158, 149, 255);
border-radius:5px;
*/
background-color: rgba(32, 125, 255,0);</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>login_2</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>sendmessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>-62</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>returnConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>-37</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendmessage()</slot>
  <slot>returnConnect()</slot>
 </slots>
</ui>
